lib/DDG/Spice/SalesTax.pm

```diff
@@ -16,15 +16,18 @@ my $US = new Locale::SubCountry("US");
 # Handle statement
 handle remainder_lc => sub {
 
-    return unless $loc && $loc->{country_code} eq 'US';
+# No need to check loc, as this IA will run for users of USA only. I am still keeping commented out code as FYI.
```

Comment:
@ajainy I think you still misunderstand -- we're not limiting this IA to US only, it should be available to everyone. The check for `$loc` was previously limiting it to US only. Now that it's been uncommented, this IA will work for everyone, which is expected.